.github/workflows/continuous-integration.yaml

```diff
@@ -35,15 +35,15 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v3
       with:
-        go-version: 1.18
+        go-version: 1.19
     - name: Check out code into the Go module directory
       uses: actions/checkout@v3
 
     - name: Lint
       uses: golangci/golangci-lint-action@v3
       with:
         args: -v
-        version: v1.45
+        version: v1.47
```

Comment:
Go 1.19 support for golangci https://github.com/golangci/golangci-lint/pull/3037

Reply:
I cannot get the linter to work. see as well https://github.com/golangci/golangci-lint/issues/2374#issuecomment-1203115918